src/classes/TableCell.ts

```diff
@@ -0,0 +1,7 @@
+import  { LocationType } from "@/classes/LocationType";
+
+export interface TableCell {
```

Comment:
Naam van de class is generic, maar eigenlijk alleen vaan toepassing op de location table.
suggestie: LocationTableCell

Reply:
zou de class ook in een ander mapje zetten. helpers maybe?